src/python/gudhi/datasets/remote.py

```diff
@@ -204,20 +241,85 @@ def fetch_bunny(file_path = None, accept_license = False):
     """
 
     file_url = "https://raw.githubusercontent.com/GUDHI/gudhi-data/main/points/bunny/bunny.npy"
-    file_checksum = 'f382482fd89df8d6444152dc8fd454444fe597581b193fd139725a85af4a6c6e'
+    file_checksum = "f382482fd89df8d6444152dc8fd454444fe597581b193fd139725a85af4a6c6e"
     license_url = "https://raw.githubusercontent.com/GUDHI/gudhi-data/main/points/bunny/bunny.LICENSE"
-    license_checksum = 'b763dbe1b2fc6015d05cbf7bcc686412a2eb100a1f2220296e3b4a644c69633a'
+    license_checksum = "b763dbe1b2fc6015d05cbf7bcc686412a2eb100a1f2220296e3b4a644c69633a"
 
     archive_path = _get_archive_path(file_path, "points/bunny/bunny.npy")
 
     if not exists(archive_path):
         _fetch_remote(file_url, archive_path, file_checksum)
         license_path = join(split(archive_path)[0], "bunny.LICENSE")
-        _fetch_remote(license_url, license_path, license_checksum)
-        # Print license terms unless accept_license is set to True
-        if not accept_license:
-            if exists(license_path):
-                with open(license_path, 'r') as f:
-                    print(f.read())
-
-    return np.load(archive_path, mmap_mode='r')
+        _fetch_remote_license(license_url, license_path, license_checksum, accept_license)
+
+    return np.load(archive_path, mmap_mode="r")
+
+
+_activities_license_url = "https://raw.githubusercontent.com/GUDHI/gudhi-data/main/points/activities/activities.LICENSE"
+_activities_license_checksum = "f5ce6749fa9d5359d7b0c4c37d0b61e5d9520f9494cd53be94295d3967ee4023"
+
+
+def fetch_daily_activities(file_path=None, subset="all", accept_license=False):
+    """
+    Load a subset of the Daily and Sports Activities dataset. This dataset comes from
+    https://archive.ics.uci.edu/ml/datasets/daily+and+sports+activities (CC BY 4.0 license).
+
+    Note that if the dataset already exists in the target location, it is not downloaded again,
+    and the corresponding dataset is read (compressed) from cache.
+
+    Parameters
+    ----------
+    file_path : string
+        Full path of the downloaded file including filename.
+
+        Default is None, meaning that it's set to "data_home/points/activities/activities_p1_left_leg.npy".
+        In this case, the LICENSE file would be downloaded as "data_home/points/activities/activities.LICENSE".
+
+        The "data_home" directory is set by default to "~/gudhi_data",
+        unless the 'GUDHI_DATA' environment variable is set.
+
+    subset : string
+        As the dataset is quite a big file to download, when specified, this argument allows to download the following subsets:
```

Comment:
Is this PR up to date? This does not seem to match the behavior of the code below.